recipes/boostdep/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class BoostDepConan(ConanFile):
+    name = "boostdep"
+    settings = "os", "arch", "compiler", "build_type"
+    description = "A tool to create Boost module dependency reports"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/boostorg/boostdep"
+    license = "BSL-1.0"
+    topics = ("conan", "boostdep", "dependency", "tree")
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def requirements(self):
+        self.requires("boost/1.74.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version][0])
+        os.rename("boostdep-boost-{}".format(self.version), self._source_subfolder)
+        license_info = self.conan_data["sources"][self.version][1]
+        tools.download(filename=os.path.basename(license_info["url"]), **license_info)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["Boost_USE_STATIC_LIBS"] = not self.options["boost"].shared
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        if self.settings.build_type != "Release":
+            raise ConanInvalidConfiguration("Only Release built_type supported")
```

Comment:
```suggestion
```

Is there a reason to remove `Release` builds? We finally [decided to keep the `Debug` builds](https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-installer-packages-remove-some-settings-from-their-package-id) just in case someone wants to debug those binaries.

Reply:
No this was done just reducing number of builds.
I will modify the recipe to make it look like other installer recipes.